src/extractor/extractor.cpp

```diff
@@ -118,7 +118,7 @@ int Extractor::run(ScriptingEnvironment &scripting_environment)
     TIMER_START(extracting);
 
     const unsigned recommended_num_threads = tbb::task_scheduler_init::default_num_threads();
-    const auto number_of_threads = std::min(recommended_num_threads, config.requested_num_threads);
+    const auto number_of_threads = std::min(recommended_num_threads, std::max(config.requested_num_threads, 1u));
```

Comment:
Yes probably overlooked or for some other historic reasons.

As I said, all the tools and the library interfaces should be consistent when possible.